---
content_type: page
description: This section includes a course description and grading information.
hide_download: true
hide_download_original: null
learning_resource_types: []
ocw_type: CourseSection
title: Syllabus
uid: bc823691-0251-c74e-875b-6f05c8f05efc
---

Course Meeting Times
--------------------

Sessions: 1 session / week, 2 hours / session

Course Overview
---------------

This seminar course will investigate the current state of computational challenges in personal genomics. We will work together to study seminal papers in the current literature, apply existing methods to complete genomes or genotypes, and explore open problems and existing research directions. The ultimate goal is understanding what can be learned computationally from a personal genome and more generally how sequence differences between individuals lead to phenotypic differences in gene expression, disease predisposition, or response to treatment.

Each week, the class will explore a different aspect of computational genomics. Students will read the two to three papers in advance of class, submit their impressions the night before, and participate in the discussion during class. Students will sign up to lead the discussion for one of topics of your choice. For each of the topics, the presenter will have a mentor with whom they will work in advance of their presentation to help prepare, understand the topic and materials, and run through the accompanying lab in advance. During the presentation, in the first 15–20 minutes the presenter will give a broader overview of the topic, then the class will spend spend 60 minutes discussing the papers in detail. The last 10 to 15 minutes of class will be reserved for students to read through the mini-lab and familiarize themselves with the datasets.

Prerequisites
-------------

This graduate level special topics course requires that students receive permission from the instructor to enroll.

Paper Reading and Participation
-------------------------------

Each week the class will read one to two articles.

The Reading phase for each paper will include, only paragraph-level comments and questions will be posted on the course site, not general comments for the paper.

The Discussion phase will include general comments on each paper will be accepted. Please use the {{% resource_link 1eb571ea-59a0-3a5e-2f1f-d27e756c849c "document template (PDF)" %}} for submitting your comments to the papers. The students leading the discussion will then have the weekend to incorporate comments from the class into their presentation for the following week.

Weekly Mini Labs
----------------

After each paper discussion, the course staff will hold a short demo on how to apply the learned methods to existing datasets. The staff will help students get set up during class, and programming assignments will help students continue these labs after class.

Term Project
------------

Each student will complete a final project that is planned out across the whole term. It should focus in depth on one or more of the topics discussed in the class. Students may either work alone or with one partner. Teams will be expected to undertake more ambitious projects. The most rewarding project topics are usually the most challenging (and possibly riskiest!). This might involve defining a biological problem, identifying relevant datasets, designing and implementing new algorithms, applying the methods, and interpreting the results. Alternatively, students can select a less risky project such as comparing several computational biology algorithms for solving the same problem and implementing, applying, and rigorously evaluating the results. Students might also analyze a relevant conference or journal article, including criticism, corrections, and / or improvements. An element of the final project grading will depend on the challenge and originality of your project, so spend enough time to choose carefully. Project proposals will be due in Session 7, and the final projects are due the night before Session 12, with in-class presentations during Session 12. The goal of the proposal is to get students thinking about their project and to make sure it is appropriate both in its subject matter and its complexity. The goal of the final project is to get students to dive more in depth into one of the topics of the course towards independent research in personal genomics.

Textbooks
---------

There are no required textbooks for the course. There are some resources available that may help as an independent outside source for any points that need clarification:

*   Foulkes, Andrea S. _Applied Statistical Genetics with R: For Population-based Association Studies_. Springer, 2009. ISBN: 9780387895536. \[Preview with [Google Books](http://books.google.com/books?id=gwW8jArHnwwC&pg=PAfrontcover)\] (available as an E-Book)
*   Hartl, Daniel, and Andrew G. Clark. _Principles of Population Genetics_. Sinauer Associates, Incorporation, 2006. ISBN: 9780878933082. (3 copies in the MIT library)

Topics
------

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
SES #
{{< thclose >}}
{{< thopen >}}
Topics
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
**Human Variation:** Challenges in human genomics, Variation, haplotypes, LD, Common and rare variant association
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
**Trait Association:** Common and rare variant association, Mendelian vs. Complex, Basics of an association study, Phenotypes
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
**Personal Genomics and Ethics:** Research studies: Consent, privacy, returning results, Medical applications: Testing, public health issues, obligations to relatives, privacy, Consumer options: Carrier testing, newborn testing, genetic testing, Gene editing: Beneﬁts, ethical issues, where to draw the line
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
**Interpreting Coding (Rare) Variants:** Rare variant detection from exome sequencing in family studies, Aggregate analyses (e.g. ExAC), Classifying function (PolyPhen), Overlapping constraint
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
**Interpreting Non-Coding (Common) Variants:** Enriched cell types, Causal variants: Epigenomics, Comparative Genomics, Target genes: HiC, genetic links, activity links, Upstream regulators: Regulatory motifs, TF binding
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
**Bayesian Fine-Mapping and Multi-dimensional GWAS:** Fine-mapping (Posterior probabilities). Bayesian models for computational ﬁne-mapping, Multi-dimensional GWAS (Multi-Phenotype, Multi-ancestry, Multi-variant)
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
**Intermediate Phenotypes and QTLs:** eQTLs, meQTLs, and other molecular traits, Allelic activity, Causality and mediation analysis, Mendelian randomization analysis
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
**Heritability and Phenotype Prediction:** Missing heritability, heritability estimation, heritability partitioning, Polygenic risk scores, Predicting intermediate phenotypes
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
**Human Ancestry and Population Genetics:** Coalescent theory. Recent Selection, Sweeps, Selection Pressures, Population Stratiﬁcation. Admixture Mapping, Ancestral genome sequencing and analysis
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
**Cancer Genomics and Single-Cell Genomics:** Genetic alterations: Somatic mutations, mutational signatures, rearrangements, Epigenetic alterations: Methylation, miRNAs, reprogramming, origin, Convergence at the regulatory region, gene, pathway, cellular level. Guiding convergence with physical, genetic, and activity networks
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
**Experimental Manipulations:** Genome editing and CRISPR, Experiment multiplexing, Screening and selection
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
**Final presentations**
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}

Grading
-------

Grades in this course will be based on the following:

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
ACTIVITIES
{{< thclose >}}
{{< thopen >}}
PERCENTAGES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
Participation
{{< tdclose >}}
{{< tdopen >}}
20%
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Mini labs
{{< tdclose >}}
{{< tdopen >}}
20%
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Presentation of assigned paper
{{< tdclose >}}
{{< tdopen >}}
30%
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
Term project
{{< tdclose >}}
{{< tdopen >}}
30%
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}